streams/src/main/java/org/apache/kafka/streams/state/RocksDBConfigSetter.java

```diff
@@ -37,4 +37,20 @@
      * @param configs       the configuration supplied to {@link org.apache.kafka.streams.StreamsConfig}
      */
     void setConfig(final String storeName, final Options options, final Map<String, Object> configs);
+
+    /**
+     * This method should close any user-constructed objects that inherit from {@code org.rocksdb.RocksObject}
```

Comment:
I would rather write the one line description in imperative.
```
Close any user-constructed objects that inherit from {@code org.rocksdb.RocksObject}.
```

Reply:
Ack